slices/xdg-user-dirs.yaml

```diff
@@ -0,0 +1,24 @@
+package: xdg-user-dirs
+
+essential:
+  - xdg-user-dirs_copyright
+
+slices:
+  config:
+    essential:
+      - libc6_libs
+    contents:
+      /etc/xdg/autostart/xdg-user-dirs.desktop:
+      /etc/xdg/user-dirs.conf:
+      /etc/xdg/user-dirs.defaults:
+
+  scripts:
```

Comment:
We typically call these `bins`.

Reply:
It was specifically requested to use `scripts` instead